tests/functional/ext/check_elif/check_elif.txt

```diff
@@ -1,2 +1,4 @@
-else-if-used:10:8:my_function:"Consider using ""elif"" instead of ""else if""":HIGH
-else-if-used:22:20:my_function:"Consider using ""elif"" instead of ""else if""":HIGH
+else-if-used:12:8:my_function:"Consider using ""elif"" instead of ""else if""":HIGH
```

Comment:
These will become `UNDEFINED` with the new updater. Do we want to change that already here?

Reply:
Let's start adding the confidence :)

Reply:
b2b9545a